chainer/testing/function.py

```diff
@@ -296,6 +306,12 @@ class FunctionTestCase(FunctionTestBase, unittest.TestCase):
         ``input_template`` is a tuple of template arrays. The returned arrays
         are expected to have the same shapes and dtypes as the template arrays.
 
+    ``check_forward_outputs(self, outputs, expected_outputs)``
+        Implements check logic of forward outputs. Typically additional check
+        can be done after calling ``super().check_forward_outputs``.
+        ``outputs`` and ``expected_outputs`` are tuples of arrays.
+        In case the check fails, ``FunctionTestCase`` should be raised.
```

Comment:
Can you fix the documentation?
```suggestion
        In case the check fails, ``FunctionTestError`` should be raised.
```

Reply:
Fixed.